plugins/teams/skills/coderabbit-adoption/coderabbit_adoption.py

```
@@ -0,0 +1,201 @@
+# GitHub search API allows 30 requests/minute for authenticated users.
+# 2-second sleep keeps us at ~30 req/min with headroom.
+SEARCH_API_SLEEP = 2
+
+
+def gh_api_get(endpoint, params=None):
+    """Call GitHub API via gh CLI and return parsed JSON."""
+    cmd = ["gh", "api", "-X", "GET", endpoint]
+    if params:
+        for key, value in params.items():
+            cmd.extend(["-f", f"{key}={value}"])
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)